tests/list_command_test.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+from unittest.mock import patch, MagicMock,Mock
+from pieces.commands.commands_functions import list_assets,startup
+from pieces.commands import commands_functions
+from io import StringIO
+import sys
+
+class TestListCommand(unittest.TestCase):
+    def test_list_command(self):
+        startup()
+        for i in ["models","assets","apps"]:
+            # Redirect stdout to a buffer
+            stdout = sys.stdout
+            sys.stdout = StringIO()
+
+            # Call the function that prints to stdout
+            list_assets(i)
```

Comment:
not going to lie this is a bit funky that list_assets is the function, but when we can pass in ("models","assets","apps") to list other materials in here as well, I know you didnt write this, but would you mind adding a github issue to split these up into their own functions or to rename this to like a generic list command that will require a material as the first parameter